### PR TITLE
paso a producción

### PR DESCRIPTION
**Pull request by miguelramirezv**

### CODE REVIEW

File: spec/controllers/pull_requests_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    let!(:pr1) { create(:pull_request, repository: repo1) }
+    let!(:pr2) { create(:pull_request, repository: repo2) }
+    let!(:user1) { create(:github_user, login: "user_login") }
+    let!(:github_session) { instance_double("GithubSession") }
+
+    before do
+      allow(github_session).to receive(:token).and_return("token")
+      allow(github_session).to receive(:user).and_return(user1)
+      expect(subject).to receive(:authenticate_github_user).and_return(true)
+      allow(subject).to receive(:github_session).and_return(github_session)
```
Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:18]: Name your test subject if you need to reference it explicitly.

File: spec/controllers/pull_requests_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    let(:repo2) { create(:repository, organization: organization2) }
+    let!(:pr1) { create(:pull_request, repository: repo1) }
+    let!(:pr2) { create(:pull_request, repository: repo2) }
+    let!(:user1) { create(:github_user, login: "user_login") }
+    let!(:github_session) { instance_double("GithubSession") }
+
+    before do
+      allow(github_session).to receive(:token).and_return("token")
+      allow(github_session).to receive(:user).and_return(user1)
+      expect(subject).to receive(:authenticate_github_user).and_return(true)
```
Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:17]: Do not use expect in before hook<br>Name your test subject if you need to reference it explicitly.<br>Prefer have_received for setting message expectations. Setup subject as a spy using allow or instance_spy.

File: spec/controllers/api/v1/likes_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    let(:pr2) { create(:pull_request, repository: repo2) }
+    let(:user1) { create(:github_user, login: "user_login") }
+    let!(:user2) { create(:github_user, login: "user_login2") }
+    let!(:like1) { create(:like, github_user: user1, likeable: pr1) }
+    let(:github_session) { instance_double("GithubSession") }
+
+    before do
+      allow(github_session).to receive(:token).and_return("token")
+      allow(github_session).to receive(:user).and_return(user1)
+      allow(subject).to receive(:github_session).and_return(github_session)
```
Comment (monkeyci) [inline spec/controllers/api/v1/likes_controller_spec.rb:19]: Name your test subject if you need to reference it explicitly.

File: spec/controllers/api/v1/froggo_teams_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+      before do
+        post :create, params: { name: "test_name", organization_id: platanus_org.id }, format: :json
+      end
+
+      it { expect(response).to have_http_status(:bad_request) }
+    end
+
+    context "when everything is ok" do
+      before do
+        post :create, params: { name: "name", organization_id: platanus_org.id, new_members_ids: [user.id] }, format: :json
```
Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:55]: Line is too long. [123/100]

File: spec/controllers/api/v1/froggo_teams_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+  before do
+    allow(github_session).to receive(:token).and_return("token")
+    allow(github_session).to receive(:organizations).and_return(
+      [{ id: platanus_org.gh_id, role: "member" }]
+    )
+    allow(github_session).to receive(:user).and_return(user)
+
+    expect(subject).to receive(:authenticate_github_user).and_return(true)
+    allow(subject).to receive(:github_session).and_return(github_session)
```
Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:22]: Name your test subject if you need to reference it explicitly.

File: spec/controllers/api/v1/froggo_teams_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+  let(:github_session) { instance_double("GithubSession") }
+
+  before do
+    allow(github_session).to receive(:token).and_return("token")
+    allow(github_session).to receive(:organizations).and_return(
+      [{ id: platanus_org.gh_id, role: "member" }]
+    )
+    allow(github_session).to receive(:user).and_return(user)
+
+    expect(subject).to receive(:authenticate_github_user).and_return(true)
```
Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:21]: Do not use expect in before hook<br>Name your test subject if you need to reference it explicitly.<br>Prefer have_received for setting message expectations. Setup subject as a spy using allow or instance_spy.

File: app/models/pull_request.rb [hunk 7]
```diff
@@ -13,7 +14,7 @@ class PullRequest < ApplicationRecord
 
   scope :within_month_limit, -> do
     where('pull_requests.gh_updated_at > ?',
-      Time.current - ENV['PULL_REQUEST_MONTH_LIMIT'].to_i.months)
+          Time.current - ENV['PULL_REQUEST_MONTH_LIMIT'].to_i.months)
```
Comment (monkeyci) [inline app/models/pull_request.rb:17]: Use one level of indentation for parameters following the first line of a multi-line method call.

File: app/javascript/store/modules/froggo_team/index.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+import actions from './actions';
```
Comment (monkeyci) [inline app/javascript/store/modules/froggo_team/index.js:1]: Definition for rule 'vue/max-len' was not found  vue/max-len

File: app/javascript/store/modules/froggo_team/actions.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
```
Comment (monkeyci) [inline app/javascript/store/modules/froggo_team/actions.js:1]: Definition for rule 'vue/max-len' was not found  vue/max-len

File: app/javascript/mixins/showMessageMixin.js [hunk 10]
```diff
@@ -0,0 +1,12 @@
+export default {
```
Comment (monkeyci) [inline app/javascript/mixins/showMessageMixin.js:1]: Definition for rule 'vue/max-len' was not found  vue/max-len

File: spec/controllers/api/v1/froggo_teams_controller_spec.rb (hunk 6 above)
Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:21]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/ExpectInHook: Do not use expect in before hook

Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:21]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/SubjectStub: Do not stub methods of the object under test.

Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:21]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.

Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:21]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/MessageSpies: Prefer have_received for setting message expectations. Setup subject as a spy using allow or instance_spy.

File: spec/controllers/api/v1/froggo_teams_controller_spec.rb (hunk 5 above)
Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:22]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/SubjectStub: Do not stub methods of the object under test.

Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:22]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.

File: spec/controllers/api/v1/froggo_teams_controller_spec.rb (hunk 4 above)
Comment (monkeyci) [inline spec/controllers/api/v1/froggo_teams_controller_spec.rb:55]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [123/100]

File: spec/controllers/api/v1/likes_controller_spec.rb (hunk 3 above)
Comment (monkeyci) [inline spec/controllers/api/v1/likes_controller_spec.rb:19]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/SubjectStub: Do not stub methods of the object under test.

Comment (monkeyci) [inline spec/controllers/api/v1/likes_controller_spec.rb:19]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.

File: spec/controllers/pull_requests_controller_spec.rb (hunk 2 above)
Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:17]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/ExpectInHook: Do not use expect in before hook

Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:17]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/SubjectStub: Do not stub methods of the object under test.

Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:17]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.

Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:17]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/MessageSpies: Prefer have_received for setting message expectations. Setup subject as a spy using allow or instance_spy.

File: spec/controllers/pull_requests_controller_spec.rb (hunk 1 above)
Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:18]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/SubjectStub: Do not stub methods of the object under test.

Comment (monkeyci) [inline spec/controllers/pull_requests_controller_spec.rb:18]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly.

File: app/javascript/store/modules/froggo_team/actions.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+      `/api/organizations/${organizationId}/froggo_teams`, decamelizeKeys({ name, newMembersIds: userIds }));
+  },
+
+  [UPDATE_FROGGO_TEAM](_, { id, name, newMembersIds, oldMembersIds }) {
+    return axios.patch(
+      `/api/froggo_teams/${id}`, decamelizeKeys({ name, newMembersIds, oldMembersIds }));
+  },
+
+  [DELETE_FROGGO_TEAM](_, { id }) {
+    const response = confirm('Estás seguro de querer borrar el equipo ?');
```
Comment (monkeyci) [inline app/javascript/store/modules/froggo_team/actions.js:21]: Unexpected confirm                               no-alert

Comment (monkeyci) [inline app/javascript/store/modules/froggo_team/actions.js:21]: Unexpected confirm                               no-alert

**Review by GabrielLyonB — APPROVED** [review]

**Pull request merged**